style.css

```diff
@@ -534,10 +560,67 @@ footer .social {
 }
 
 .dynadiv {
-  height: 30vh;
   background-color: #fff;
 }
 
+.openover {
```

Comment:
## I kindly suggest that you follow the Figma design by ensuring you use the same font, colours, images, text, margins). 

- [ ] Kindly ensure that your Figma Design is Responsive by ensuring the close X button shows in the Mobile design.
- [ ] Kindly maintain the same background color of white not transparent and ensure your body's background below the modal is blurred when the modal is open as seen in the Figma Design.
- [ ] For Mobile, Kindly ensure that the image is placed in the center and of the same height and width as the Figma Design. 
- [ ] For Desktop, kindly ensure that the image takes up 60% of width.
- [ ] Kindy ensure that the modal does not take up the full width and height of the page as seen in the Figma design.


| Your Design | Figma Design |
| ------------- | ------------- |
| <img width="150" alt="image" src="https://user-images.githubusercontent.com/52934128/197008302-08aceceb-6904-43a0-ac5c-d948045a3013.png">  | <img width="209" alt="image" src="https://user-images.githubusercontent.com/52934128/197008501-ca4d288b-a067-4369-95c3-f88b772d7c6c.png">  |
| <img width="945" alt="image" src="https://user-images.githubusercontent.com/52934128/197010590-ff35243b-7b92-4f2e-a172-e0c05a9c2866.png"> | <img width="585" alt="image" src="https://user-images.githubusercontent.com/52934128/197010497-d1cea147-c116-44c7-b0d7-74605803a52e.png">  |